### ISSUE TITLE
Bot API 5.0

### ISSUE THREAD

**Issue by Syfaro** [go-telegram-bot-api/telegram-bot-api#390]
New methods
- [ ] `logOut`
- [ ] `close`
- [ ] `unpinAllChatMessages`
- [ ] `copyMessage`

Added parameters
- [ ] `ip_address` in `setWebhook`
- [ ] `drop_pending_updates` in `setWebhook`, `deleteWebhook`
- [ ] `only_if_banned` in `unbanChatMember`
- [ ] `disable_content_type_detection` in `sendDocument`
- [ ] `message_id` in `unpinChatMessage`
- [ ] `heading` in `sendLocation`, `editMessageLiveLocation`
- [ ] `proximity_alert_distance` in `sendLocation`, `editMessageLiveLocation`
- [ ] `horizontal_accuracy` in `sendLocation`, `editMessageLiveLocation`
- [ ] `is_anonymous` in `promoteChatMember`
- [ ] `entities` or `caption_entities` in `sendMessage`, `sendPhoto`, `sendVideo`, `sendAnimation`, `sendAudio`, `sendDocument`, `sendVoice`, `sendPoll`, `editMessageText`, `editMessageCaption`
- [ ] `google_place_id`, `google_place_type` in `sendVenue`
- [ ] `allow_sending_without_reply` in `sendMessage`, `sendPhoto`, `sendVideo`, `sendAnimation`, `sendAudio`, `sendDocument`, `sendSticker`, `sendVideoNote`, `sendVoice`, `sendLocation`, `sendVenue`, `sendContact`, `sendPoll`, `sendDice`, `sendInvoice`, `sendGame`, `sendMediaGroup`

Added fields
- [ ] `ip_address` in `WebhookInfo`
- [ ] `bio`, `linked_chat_id`, `location` in `Chat`
- [ ] `file_name` in `Audio`, `Video`
- [ ] `disable_content_type_detection` in `inputMediaDocument`
- [ ] `live_period` in `Location`
- [ ] `heading` in `Location`, `InlineQueryResultLocation`, `InputLocationMessageContent`
- [ ] `proximity_alert_distance` in `Location`, `InlineQueryResultLocation`, `InputLocationMessageContent`
- [ ] `proximity_alert_triggered` in `Message`
- [ ] `horizontal_accuracy` in `Location`, `InlineQueryResultLocation`, `InputLocationMessageContent`
- [ ] `sender_chat` in `Message`
- [ ] `is_anonymous` in `ChatMember`
- [ ] `entities` or `caption_entities` in `InputMedia`, `InlineQueryResult`, `InputTextMessageContent` types
- [ ] `google_place_id`, `google_place_type` in `Venue`, `InlineQueryResultVenue`, `InputVenueMessageContent`

Added types
- [ ] `ProximityAlertTriggered`

Miscellaneous
- [ ] Support audio and document albums with `sendMediaGroup` 
- [ ] Football, slot machine emoji in `sendDice`

**Comment by HirbodBehnam** [comment]
Hello
Is there any plan for adding support of multiple file uploads in version 5.0?

**Comment by Syfaro** [comment]
@HirbodBehnam yes, it's tagged on the v5 milestone. You can follow it more on #356.
